content/en/synthetics/guide/_index.md

```diff
@@ -1,38 +1,53 @@
 ---
 title: Synthetic Monitoring Guides
 kind: guide
+aliases:
+  - /synthetics/faq/clone-test/
+  - /synthetics/faq/uptime-check-internal-website/ 
```

Comment:
the aliases for these should go inside the new guide files, not this landing page.

Reply:
I applied the FAQ alias to the Clone Test guide but we did not carry over the Uptime Check FAQ.